pom.xml

```diff
@@ -49,13 +49,8 @@
         <commons-collections4.version>4.1</commons-collections4.version>
         <commons-lang3.version>3.9</commons-lang3.version>
         <ddf.jaxb.version>1.0.1</ddf.jaxb.version>
-        <!--
-        NOTE: ddf-sdk.version should be removed if itests no longer require the sdk
-        Most likely DDF 2.22.0
-         -->
-        <ddf-sdk.version>2.19.0</ddf-sdk.version>
         <ddf.support.version>2.3.16</ddf.support.version>
-        <ddf.version>2.21.0</ddf.version>
+        <ddf.version>2.25.2-SNAPSHOT</ddf.version>
```

Comment:
❓Is this considered a breaking change? Should the major or minor of this project be bumped in this PR?